src/server/commands.rs

```diff
@@ -274,7 +276,9 @@ impl Command {
             }
             "PASS" => {
                 let password = parse_to_eol(cmd_params)?;
-                Command::Pass { password }
+                Command::Pass {
+                    password: Password::new(password),
```

Comment:
Do you need to specify the field name?

Reply:
I do, since this does not apply anymore: https://doc.rust-lang.org/stable/book/ch05-01-defining-structs.html#using-the-field-init-shorthand-when-variables-and-fields-have-the-same-name